okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -161,6 +165,9 @@
   @GuardedBy("lock")
   private LinkedList<PendingStream> pendingStreams = new LinkedList<PendingStream>();
   private final ConnectionSpec connectionSpec;
+  private FrameWriter rawFrameWriter;
```

Comment:
Partially done, renamed the FrameWriter for test, but I don't see the point of sending the connection preface and settings under the lock.